docs/commons/FAIR/Assessment-Recommended-API.rst

```diff
@@ -76,3 +76,22 @@ All post requests must submit a body with the resource to assess as follows:
    * - ``/assess/algorithm/``
      - ``algoid`` and ``resource_identifier``
      - Test result in JSON-LD
+
+In some cases, FAIR assessments may require inspecting multiple large resources. This is, for example, the case when assessing Research Objects using the **FAIROs** tool. In such situations, the API may not immediately return the test results in JSON-LD format. Instead, it returns a JSON response with the following information:
```

Comment:
I would not mention the FAIROs tool here. I would just state that this is something that may happen and the adopted solution